mythril/laser/ethereum/taint_analysis.py

```diff
@@ -107,7 +107,8 @@ def execute(statespace, node, state, initial_stack=[]):
             records = TaintRunner.execute_node(node, record, index)
 
             result.add_records(records)
-
+            if len(records) == 0:          # continue if there is no record to work on
```

Comment:
Records should always be non empty here right? Why is that not the case here?

Reply:
If a node doesn’t have any states in that case it might not have any record to work on right?